modules/core/shared/src/main/scala/retry/package.scala

```diff
@@ -6,9 +6,11 @@ import cats.syntax.flatMap._
 import scala.concurrent.duration.FiniteDuration
 
 package object retry {
-  def retryingM[A] = new RetryingPartiallyApplied[A]
+  @deprecated("Use retryingOnFailures instead", "")
```

Comment:
I left the `since` parameter as a blank string as I'm not sure what the version of this will be for the release.

Reply:
We'll release this and your other PR as 2.1.0